pyramid_openapi3/tests/test_app_construction.py

```diff
@@ -39,17 +39,30 @@
 """
 
 SPLIT_DOCUMENT = b"""
-    openapi: "3.0.0"
+    openapi: "3.1.0"
     info:
       version: "1.0.0"
       title: Foo API
     servers:
       - url: /api/v1
     paths:
       /foo:
-        $ref: "paths.yaml#/foo"
+        # $ref: "paths.yaml#/foo"
```

Comment:
Can't get `refs` to work. Needs further investigation.

Reply:
So it seems `refs` work for everything but for paths? 

Details: https://github.com/Pylons/pyramid_openapi3/pull/206/commits/9b080c1282358f290c04e0234f5ee6e26bd86678

Reply:
Is this good enough?

Reply:
@damonhook: can you pls have a look if you can get `examples/splitfile` to work with paths being in a separate `paths.yaml` file? I can probably take it from there then, fix the rests of the tests and such.

Reply:
https://github.com/python-openapi/openapi-spec-validator/issues/207

Reply:
I see that this `openapi-spec-validator` issue has been closed. The refs in the paths should hopefully work now

<details><summary>Example openapi.yaml</summary>
<p>

```yaml
openapi: 3.1.0

info:
  title: Example API
  version: 1.0.0

paths:
  /:
    $ref: "#/components/pathItems/rootPath"

components:
  pathItems:
    rootPath:
      get:
        responses:
          200:
            description: Example Response⏎
```

</p>
</details> 

```bash
openapi-spec-validator openapi.yaml

OK
```
Previous version of `openapi-spec-validator` was raising an error

Will give it a test with the `examples/splitfile`